### ISSUE TITLE
Should we use SVMs for FGVC? 

### ISSUE THREAD

**Issue by CupidJay** [jiangtaoxie/fast-MPN-COV#12]
Hi, I am reading about your iSQRT paper and i think it is quite interesting. However, I am confused about the usage of SVMs.
You wrote "After finetuning, the outputs of iSQRT-COV layer are ℓ2−normalized before inputted to train k one-vs-all linear SVMs with hyperparameter C = 1" in your paper but i didn't find it in your released code.

**Comment by jiangtaoxie** [comment]
@CupidJay We perform SVM classifier for fair comparison with existing methods. In fact, it improve performance very marginal (<=0.1%). So SVM is not a required option.

**Comment by CupidJay** [comment]
Thanks for your kind reply.
